patches/@segment+sovran-react-native+1.0.1.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/node_modules/@segment/sovran-react-native/lib/typescript/src/index.d.ts b/node_modules/@segment/sovran-react-native/lib/typescript/src/index.d.ts
+index 961c442..aba4c3a 100644
+--- a/node_modules/@segment/sovran-react-native/lib/typescript/src/index.d.ts
++++ b/node_modules/@segment/sovran-react-native/lib/typescript/src/index.d.ts
+@@ -1,3 +1,4 @@
+-export { createStore, type Store, type Notify, type Unsubscribe, } from './store';
```

Comment:
TODO: This file can be removed when we upgrade to Typescript 4.4.X or higher. 
It was causing an error similar to this with our current typescript version https://github.com/segmentio/analytics-react-native/issues/607

Reply:
Maybe we should add this as a comment in the patch so we'll know more easily?